test/manual/amp-mega-menu.amp.html

```diff
@@ -0,0 +1,138 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-mega-menu</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    #title {
+      text-align: center;
+      font-size: 25px;
+      padding: 15px;
+    }
+    header > * {
+      background: white;
+    }
+    .heading, .content, .section {
+      padding: 10px;
+      margin: 0;
+    }
+    .section {
+      display: inline-block;
+      vertical-align: top;
+    }
+    ul {
+      list-style-type: none;
+    }
+    amp-embed {
+      float: right;
+      margin-right: 50px;
+    }
+    fieldset {
+      border: none;
+    }
+    #main {
+      background: #ccc;
+      font-size: 18px;
+    }
+  </style>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-mega-menu" src="https://cdn.ampproject.org/v0/amp-mega-menu-0.1.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-element="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+  <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
+  <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
+  <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+</head>
+<body>
+  <header>
+    <div id="title">amp-mega-menu</div>
+
+    <amp-mega-menu layout="fixed-height" height="55">
+      <amp-list layout="fixed-height" height="600" src="/list/ecommerce-nested-menu" items="." single-item>
```

Comment:
It does feel a bit out of place, though I'm leaning towards keeping the json here for now since it will be used by multiple manual tests and possibly examples. @cathyxz

Reply:
There are two options. One is to keep it in the routes file like you currently have. The other is to add a static `.json` file to a local folder (e.g. you can create a folder within `test/manual` for `amp-mega-menu`). For example within the set of amp-list examples, we have `test/manual/amp-list/amp-list-data.json`. 

I think that so long is the JSON blob isn't ridiculously large, this is fine.